### DISCUSSION TITLE
TypeError: 'ABCMeta' object is not iterable

### DISCUSSION THREAD

**Discussion by br-jisc** [agronholm/sqlacodegen#278] [General]
Just installed sqlacodegen 2.3.0 and I always get the error TypeError: 'ABCMeta' object is not iterable.

This is running on Python 3.10.0a7.

Is there some change I can make to my Python environment to get sqlacodegen to run?

**Comment by agronholm** [comment]
1. Try the latest 3.0 release candidate
2. Use a release version of Python, not an alpha
